x-pack/plugins/rule_registry/server/utils/create_lifecycle_executor.test.ts

```diff
@@ -128,12 +130,16 @@ describe('createLifecycleExecutor', () => {
           {
             fields: {
               [ALERT_ID]: 'TEST_ALERT_0',
+              [OWNER]: 'CONSUMER',
```

Comment:
Should we update these to match so we no longer use `OWNER` and instead `CONSUMER`?

Reply:
renaming this field should happen in another PR, but yes we do want to change `OWNER` to `CONSUMER`.

Reply:
fyi: https://github.com/elastic/kibana/pull/107857